go/ql/test/query-tests/Security/CWE-918/RequestForgery.expected

```diff
@@ -32,9 +26,6 @@ nodes
 | tst.go:24:66:24:72 | tainted | semmle.label | tainted |
 | tst.go:27:11:27:29 | ...+... | semmle.label | ...+... |
 | tst.go:29:11:29:40 | ...+... | semmle.label | ...+... |
-| tst.go:35:2:35:2 | definition of u [pointer] | semmle.label | definition of u [pointer] |
-| tst.go:36:2:36:2 | implicit dereference | semmle.label | implicit dereference |
-| tst.go:36:2:36:2 | u [pointer] | semmle.label | u [pointer] |
```

Comment:
Why do these three nodes not appear in the test output any more?

Reply:
That's a good question. It was the only such change across all our tests. But all the actual results are preserved, so I'm not sure it's worth it to investigate too much.